documentation/sphinx/source/api-python.rst

```diff
@@ -598,7 +598,11 @@ Writing data
 
 .. method:: Transaction.clear_range_startswith(prefix)
 
-    Removes all the keys ``k`` such that ``k.startswith(prefix)``, and their associated values. |immediate-return|
+    Removes all keys ``k`` such that ``begin <= k < end``, and their associated values. |immediate-return|
```

Comment:
It looks like this comment was intended for `clear_range` but has been applied to `clear_range_startswith`. It's reasonable to add the new comment to both; my recommendation would be to add a blurb to api-common.rst.inc and then use that same blurb in all functions that require it in api-ruby, api-python, and api-c.